docs/setup/upgrade.asciidoc

```diff
@@ -44,13 +44,12 @@ a|
 [[upgrade-before-you-begin]]
 === Before you begin
 
-WARNING: {kib} automatically runs upgrade migrations when required. To roll back to an earlier version in case of an upgrade failure, you **must** have a {ref}/snapshot-restore.html[backup snapshot] available. This snapshot must include the `kibana` feature state or all `kibana*` indices. For more information see <<upgrade-migrations, upgrade migrations>>.
+WARNING: {kib} automatically runs upgrade migrations when required. To roll back to an earlier version in case of an upgrade failure, you **must** have a {ref}/snapshot-restore.html[backup snapshot] available. This snapshot must include the `kibana` feature state. For more information see <<upgrade-migrations, upgrade migrations>>.
```

Comment:
`include_global_state: true` will also work, and defaults to `true`. I'm not sure how to add that here without making the note huge or being less than clear, but that is generally how we expect folks to use it - just leave the default of including global state so you can be sure your configuration is backed up, and I don't want to imply that you *have to* use the `feature_states` parameter now (as I expect most users won't need to).

I don't want to block on this if we just want to make sure the docs are correct before the holidays, though.

Reply:
Thanks for the feedback. In https://github.com/elastic/kibana/pull/120174/commits/1cffe209696d4546363ebbd577198070a32d2d15, I revised this to:

> To roll back to an earlier version in case of an upgrade failure, you **must** have a backup snapshot that includes the `kibana` feature state. Snapshots include this feature state by default.